external_libraries/oscpack_1_1_0/osc/OscTypes.h

```diff
@@ -37,42 +37,17 @@
 #ifndef INCLUDED_OSCPACK_OSCTYPES_H
 #define INCLUDED_OSCPACK_OSCTYPES_H
 
+#include <stdint.h>

```

Comment:
good point! OK I was about to push a patch further upstream (as you rightly suggested) but I take it this particular patch is unsuitable for upstreaming. (fine for us tho)